package.json

```diff
@@ -18,6 +18,10 @@
     "concurrently": "^8.2.2",
     "create-react-app": "^5.0.1",
     "dotenv": "^16.4.4",
+    "mongoose": "^8.1.2"
+  },
+  "devDependencies": {
+    "gh-pages": "^6.1.1"
```

Comment:
I think we can get rid of the gh-pages dependency since we're using vercel instead now

Reply:
Sankalp took care of this.